client/src/components/Dataset/services.ts

```diff
@@ -0,0 +1,79 @@
+import { fetcher } from "schema";
+import { safePath } from "utils/redirect";
+import type { paths } from "schema";
+import type { FetchArgType } from "openapi-typescript-fetch";
```

Comment:
```suggestion
import { safePath } from "utils/redirect";
import type { FetchArgType } from "openapi-typescript-fetch";
```

Reply:
Thanks, interesting that no linter caught that!

Reply:
I need to look at why that didn't get caught.  There was another one here that I ran into, as well: https://github.com/galaxyproject/galaxy/pull/15116/commits/8af76ab49f56c39b5791a88d887427ad345cdeef